src/lib/jsonParse/jsonParse.cpp

```diff
@@ -428,6 +432,13 @@ std::string jsonParse
   ptree              tree;
   ptree              subtree;
   std::string        path;
+  struct timespec    start;
+  struct timespec    end;
+
+  if (reqTimeStatistics)
```

Comment:
Not sure what you mean here ...
If I understand correctly, it seems like you're thinkiung of a typical SHELL construction.
Doesn't work like that in C